src/std/env.ab

```diff
@@ -15,59 +16,71 @@ pub fun get_env_var(var: Text, file: Text = ".env"): Text {
     return ""
 }
```

Comment:
There is a bug here on line 11. It should be:
```
if file_exist(file) {
```

Reply:
good catch!